### ISSUE TITLE
Source filter not working when dragging

### ISSUE THREAD

**Issue by pedrokost** [sporritt/jsPlumb#371]
Please see the following gif:
![ezgif-2187642604](https://cloud.githubusercontent.com/assets/725286/7591512/8df3a04e-f8ce-11e4-9e4b-a6beb9815b55.gif)

When I add a new DOM element as a graph node (this happens when you click Add), I convert it to a source/target element and make it draggable.

```javascript
// some code to add new new DOM element to convert to a node
var newNode = myJsPlumb.getSelector('.process-node:not(.jsplumb-draggable)');
myJsPlumb.makeTarget(newNode, { allowLoopback: true });  // + default options
myJsPlumb.makeSource(newNode, { filter: '.handle' }); // + default options
myJsPlumb.draggable(newNode); // + default options
```

As you can see in the GIF, the dragging is also applied to the orange square ('.handle') on this new DOM element, but not the rest. 

The drag options are these:

```javascript
DragOptions: {
  containment: true,
  stop: function (event) { //gets called on every drag
    var nodeId = event.el.getAttribute('data-node-id');
    var container = that.$('#process-flowchart-container');
    var containerWidth  = container.width();
    var containerHeight = container.height();
    var relativeX = event.pos[0] / containerWidth;
    var relativeY = event.pos[1] / containerHeight;

    console.log(nodeId, relativeX, relativeY);
    that.sendAction('nodeMoved', nodeId, relativeX, relativeY);
  }
},
```

Any idea what is going on? Is this a bug?

**Comment by sporritt** [comment]
it certainly looks pretty much ok...you're definitely getting only one element in the `newNode` selector?

**Comment by pedrokost** [comment]
Yes, the selector returns an array of only one element.

```html
<div class="process-node" id="node" data-node-id="" style="top:NaN%; left:NaN%"> RV 60-60 <div class="handle"></div> </div>
```

Is there a way to deactivate the dragging on a selection of elements after .draggable() is called? Maybe disabling and reenabling dragging would fix it.

**Comment by sporritt** [comment]
not sure what you mean...

it might be easier to reproduce in a jsfiddle?

**Comment by pedrokost** [comment]
What is meant is whether there is a way to remove the drag-drop event handlers from all the nodes? Somethink like `$(allwindows).resetdraggable();`.

I will try to create a jsfiddle reproducing the issue.

**Comment by pedrokost** [comment]
I wasn't able to reproduce the issue in jsFiddle, but this helped me hunt down the bug. 

I am using this in combination with Ember, which automatically inserts new nodes as soon as they are created (by may have no id until they are saved). Then, as soon as the node is saved, it is assigned an id, and Ember automatically updates the DOM -> thus changing the id of the element from `node` to something like: `node123`.

I have resolved this issue by creating ids in advance, before the element is inserted into the DOM.

Closing, as it is not a jsPlumb issue.
